proxy/schemas/responses.py

```
@@ -48,27 +48,35 @@ class Tool(RootModel):
-    index: int
-    message: Message
-    finish_reason: Optional[str] = Field(
-        default=None, pattern="^(stop|length|function_call)$"
-    )
-    logprobs: Optional[LogProbs] = None
+class Reasoning(BaseModel):
+    effort: Optional[Any] = None
+    summary: Optional[str] = None
+
 
+class OutputContent(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)